docs/documentation/concepts/units-of-measurement.md

```diff
@@ -71,3 +71,117 @@ When creating QuantityType states the framework offers some useful packages and
 The `org.eclipse.smarthome.core.library.unit` package contains the classes `SIUnits`, `ImperialUnits` and `SmartHomeUnits` which provide units unique to either of the measurement systems and common units used in both systems.
 The `MetricPrefix` class provides prefixes like MILLI, CENTI, HECTO, etc. which are wrappers to create derived units.
 The `org.eclipse.smarthome.core.library.dimension` and `javax.measure.quantity` packages provide interfaces which are used to type the generic QuantityType and units. 
+
+## List of Units
+
+List of the unit available with the UoM (Unit of Measurement) types
```

Comment:
That's not real sentence currently. How about something along the lines of 

```
All units which are currently supported by default are listed in the tables below.
```

?